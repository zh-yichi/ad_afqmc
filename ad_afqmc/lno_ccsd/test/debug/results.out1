# frag  hf_orb_cr  err  ccsd_orb_cr  err  afqmc_orb_corr  err  mp2_orb_corr  ccsd_orb_corr  n_orb  n_elec  time
  1  -0.034543  0.000977  -0.000182  0.000977  -0.034725  0.000000  -0.026388    None    10  2  59.84
  2  -0.030760  None  -0.003924  None  -0.034685  None  -0.026361    None    10  2  60.68
# final results 
# mean-field energy: -2.2573139273
# hf_cr: -0.065303 +/- 0.000977
# ccsd_cr: -0.004106 +/- 0.000977
# lno_afqmc_corr: -0.069410 +/- 0.000000
# e_mp2_corr: -0.052749
# e_ccsd_corr: 0.000000
# mp2_corrected lno_afqmc_corr: -0.069500
# lno-thresh [0.0001, 1e-05]
# number of orbitals: average 10.00 maxium 10
# number of electrons: average 2.00 maxium 2
# total run time: 120.52
