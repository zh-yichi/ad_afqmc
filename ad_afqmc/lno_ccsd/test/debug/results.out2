# frag  hf_orb_cr  err  ccsd_orb_cr  err  afqmc_orb_corr  err  mp2_orb_corr  ccsd_orb_corr  n_orb  n_elec  time
  1  -0.031017  0.000944  -0.003633  0.000937  -0.034650  0.000013  -0.026429  -0.034766  14  4  91.15
  2  -0.034790  0.001065  0.000148  0.001058  -0.034643  0.000014  -0.026401  -0.034722  13  4  91.24
# final results 
# mean-field energy: -2.2573139273
# hf_cr: -0.065807 +/- 0.001423
# ccsd_cr: -0.003485 +/- 0.001413
# lno_afqmc_corr: -0.069293 +/- 0.000019
# e_mp2_corr: -0.052830
# e_ccsd_corr: -0.069488
# mp2_corrected lno_afqmc_corr: -0.069302
# lno-thresh [9.999999999999999e-06, 1e-06]
# number of orbitals: average 13.50 maxium 14
# number of electrons: average 4.00 maxium 4
# total run time: 182.39
