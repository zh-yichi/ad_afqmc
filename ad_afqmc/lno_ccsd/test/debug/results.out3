# frag  hf_orb_cr  err  olp_ratio  err  ccsd_orb_cr0  err  ccsd_orb_cr1  err  ccsd_orb_cr2  err ccsd_orb_cr  err  afqmc_orb_corr  err  mp2_orb_corr  ccsd_orb_corr  n_orb  n_elec  time
  1  -0.033485  0.000207  0.966878  0.000238  0.071635  0.000579  -0.120448  0.001027  0.047979  0.000648  -0.000834  0.000204  -0.034319  0.000006  -0.026017    None    4  18  315.11
  2  -0.033960  0.000179  0.967273  0.000208  0.072095  0.000479  -0.120886  0.000839  0.047663  0.000532  -0.001128  0.000176  -0.035089  0.000006  -0.026646    None    4  18  326.85
# final results 
# mean-field energy: -2.2573139273
# hf_cr: -0.067445 +/- 0.000274
# average olp_ratio: 0.967075 +/- 0.000224
# ccsd_cr0: 0.143730 +/- 0.000751
# ccsd_cr1: -0.241334 +/- 0.001326
# ccsd_cr2: 0.095642 +/- 0.000838
# ccsd_cr: -0.001962 +/- 0.000269
# lno_afqmc_corr: -0.069408 +/- 0.000008
# e_mp2_corr: -0.052663
# e_ccsd_corr: 0.000000
# mp2_corrected lno_afqmc_corr: -0.069584
# lno-thresh [0.0001, 1e-05]
# number of orbitals: average 4.00 maxium 4
# number of electrons: average 18.00 maxium 18
# total run time: 641.96
