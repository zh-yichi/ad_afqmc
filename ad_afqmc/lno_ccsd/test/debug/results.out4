# frag  hf_orb_cr  err  ccsd_orb_cr  err  afqmc_orb_corr  err  mp2_orb_corr  ccsd_orb_corr  n_orb  n_elec  time
  1  -0.031997  0.001898  -0.002204  0.001733  -0.034202  0.000265  -0.022115  -0.033108  10  10  49.82
  2  -0.028321  0.001277  -0.005103  0.001186  -0.033424  0.000205  -0.021855  -0.033170  10  10  109.14
  3  -0.027392  0.001998  -0.005991  0.001823  -0.033382  0.000414  -0.020898  -0.033161  10  10  53.53
  4  -0.029662  0.002565  -0.003385  0.002409  -0.033046  0.000230  -0.020898  -0.033161  10  10  49.56
  5  -0.029406  0.001472  -0.004596  0.001359  -0.034002  0.000286  -0.021855  -0.033170  10  10  50.89
# final results 
# mean-field energy: -5.2476173426
# hf_cr: -0.146778 +/- 0.004239
# ccsd_cr: -0.021279 +/- 0.003922
# lno_afqmc_corr: -0.168056 +/- 0.000647
# e_mp2_corr: -0.107621
# e_ccsd_corr: -0.165770
# mp2_corrected lno_afqmc_corr: -0.168055
# lno-thresh [1e-06, 1e-07]
# number of orbitals: average 10.00 maxium 10
# number of electrons: average 10.00 maxium 10
# total run time: 312.94
