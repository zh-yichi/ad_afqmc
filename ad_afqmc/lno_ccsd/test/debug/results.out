# frag  hf_orb_cr  err  ccsd_orb_cr  err  afqmc_orb_corr  err  mp2_orb_corr  ccsd_orb_corr  n_orb  n_elec  time
  1  -0.014577  0.000781  0.000926  0.000781  -0.013651  0.000000  -0.010183  -0.013651  3  2  30.30
  2  -0.026651  0.001764  0.000742  0.001764  -0.025909  0.000000  -0.016723  -0.025909  3  2  54.21
  3  -0.037867  0.002276  0.005342  0.002380  -0.032525  0.000129  -0.019489  -0.025909  2  3  34.67
  4  -0.031157  0.002953  -0.001432  0.003061  -0.032589  0.000207  -0.019489  -0.025909  2  3  32.65
  5  -0.020939  0.001665  -0.005958  0.001822  -0.026897  0.000170  -0.016536  -0.025909  2  3  32.55
# final results 
# mean-field energy: -5.3553816675
# hf_cr: -0.131191 +/- 0.004516
# ccsd_cr: -0.000380 +/- 0.004698
# lno_afqmc_corr: -0.131571 +/- 0.000297
# e_mp2_corr: -0.082420
# e_ccsd_corr: -0.117287
# mp2_corrected lno_afqmc_corr: -0.229598
# lno-thresh [0.01, 0.001]
# number of orbitals: average 2.40 maxium 3
# number of electrons: average 2.60 maxium 3
# total run time: 184.38
